pkgpanda/docs/http-swagger.yaml

```diff
@@ -0,0 +1,116 @@
+swagger: '2.0'
+info:
+  title: DC/OS Pkgpanda API
+  version: '1.0.0'
+basePath: /pkgpanda
+
+
+definitions:
+
+  PackageId:
+    description: A unique pkgpanda package ID. Composed of a name and a version separated by `--`.
+    type: string
+    example: 'mesos--abcdef'
+    # Pattern adapted from pkgpanda/__init__.py.
+    pattern: '/^[a-zA-Z0-9@_+]([a-zA-Z0-9@._+\-]*[a-zA-Z0-9@._+])?--[a-zA-Z0-9@_+:.]+$/'
```

Comment:
There is a much tighter pattern this is required to follow. See: https://github.com/dcos/dcos/blob/master/pkgpanda/__init__.py#L83